README.md

```diff
@@ -128,7 +135,7 @@ By default the dbp chart will deploy fully.
 To disable specific components you can set the following values to false when deploying:
 ```
 alfresco-content-services.enabled
-alfresco-process-services.enabled
+activiti-cloud-full-example.enabled
```

Comment:
This should be sufficient for this iteration, but in the future we need to confirm with Greg, George, and Marcello on whether we deploy a full example or only common APS infrastructure.

Reply:
We should discuss this.